app/utils.py

```diff
@@ -4,16 +4,35 @@
 from enum import Enum
 from types import GeneratorType
 from boardgamegeek import BGGClient, CacheBackendSqlite
-from schema import Game
+from app.schema import Game
 import datetime
 import json
+from itertools import islice
+
+
+def take(n, iterable):
+    "Return first n items of the iterable as a list"
+    return list(islice(iterable, n))
 
 
 def isoformat(o):
+    "returns input datetime in isoformat"
```

Comment:
Do you plan to change/customize this function at some point? If you think it'll stay with it's current definition, then I think you're just better off calling `o.isoformat()` instead of `isoformat(o)`…